test/Grpc.AspNetCore.Server.Tests/DefaultGrpcInterceptorActivatorTests.cs

```diff
@@ -77,7 +77,7 @@ public void GrpcInterceptorCanResolveArgumentsFromArg()
                 .Setup(sp => sp.GetService(typeof(Mutex)))
                 .Returns(mutex);
 
-            var interceptor = new DefaultGrpcInterceptorActivator<GrpcInterceptor>(mockServiceProvider.Object).Create(10) as GrpcInterceptor;
+            var interceptor = (GrpcInterceptor)new DefaultGrpcInterceptorActivator<GrpcInterceptor>(mockServiceProvider.Object).Create(10);
```

Comment:
I'm wondering if IGrpcInterceptorActivator<T> should have `T Create()` and `Release(T)` instead of just the base interface IGrpcInterceptor's `Interceptor Create()` and `Release(Interceptor )` methods?

Reply:
Because then you need to create a generic type and use reflection to call it in the call handlers.